code/b-parasite/src/prst/adc.h

```diff
@@ -16,10 +16,18 @@ typedef struct prst_adc_soil_moisture {
   double percentage;
 } prst_adc_soil_moisture_t;
 
+typedef struct prst_adc_photo_sensor {
+  int16_t raw;
+  // Should check minimum and maximum values
```

Comment:
I think a good strategy is to define:
0x0000 -> pitch black (no light at all)
0xffff -> super bright (maybe directly in the sun + a safe margin)

We can update this comment by saying "Needs calibration".

Reply:
Done